Godeps/Godeps.json

```diff
@@ -3,7 +3,8 @@
 	"GoVersion": "go1.6",
 	"GodepVersion": "v74",
 	"Packages": [
-		"./..."
+		"./...",
+		"github.com/progrium/go-extpoints"
```

Comment:
Please remove progrium. We no longer use this, though the documentation is outdated.